src/ConversationDB.php

```diff
@@ -63,7 +64,7 @@ public static function selectConversation($user_id, $chat_id, $limit = null)
 
             $results = $sth->fetchAll(\PDO::FETCH_ASSOC);
 
-        } catch (PDOException $e) {
+        } catch (\Exception $e) {
```

Comment:
No, unfortunately not :confused: 

This is the exception I get, which is not caught:
```
PDOStatement::execute(): SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`telegrambot`.`conversation`, CONSTRAINT `conversation_ibfk_1` FOREIGN KEY (`user_id`) REFERENCES `user` (`id`))
```
I checked the PDO documentation and found that the PDOException is the only exception type available, but the one being thrown is not of that type. If you can find a more specific one to catch that would be great, but I didn't find any...